test/json_test.exs

```diff
@@ -53,7 +53,16 @@ defmodule ExCoveralls.JsonTest do
       Json.execute(@source_info)
     end) =~ @stats_result
 
-    assert(File.read!(report) =~ ~s({"source_files":[{"coverage":[0,1,null,null],"name":"test/fixtures/test.ex","source":"defmodule Test do\\n  def test do\\n  end\\nend\\n"}]}))
+    assert(
```

Comment:
JSON key ordering is not guaranteed anymore in OTP 26, so just decoding it here (and below) does the trick